changelog/1.17.0.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- (?) Wsp orgs migration
+
+### Features ✨
+
+- Environment Providers **TODO**
```

Comment:
@khorne3 I kind of left this one open for you and @f0ssel to do your thing on. Sorry about that, just wasn't sure to what extent we'll list out the changes